test/extended/util/disruption/controlplane/controlplane.go

```diff
@@ -162,3 +163,38 @@ func allClusterVersionsAreGTE(version semver.Version, config *rest.Config) (bool
 	}
 	return true, nil
 }
+
+type ApiFastRestorationTest struct{}
```

Comment:
what is this supposed to do in addition to a `/readyz` check

Reply:
no, it does not
refactored the code to make it a lot simpler (and only use ``/readyz``), please have another look